test/crypto/elliptic.js

```diff
@@ -1,4 +1,8 @@
 const openpgp = typeof window !== 'undefined' && window.openpgp ? window.openpgp : require('../..');
+const elliptic_curves = require('../../src/crypto/public_key/elliptic');
+const hashMod = require('../../src/crypto/hash');
+const config = require('../../src/config');
```

Comment:
Why not access opengpg.config?

Reply:
Because I'm modifying the config below, and I want it to have effect for the `elliptic_curves` functions.
`require('../../src')` returns a different instance of openpgp than `require('../..')`.